### PR TITLE
Move permit holder type selection to task list for standard rules

### PR DESCRIPTION
**Pull request by ajaxscape**
Implements tests for: https://eaflood.atlassian.net/browse/WE-2327

### CODE REVIEW

File: features/test_error.feature [hunk 1]
```diff
@@ -6,6 +6,8 @@ Feature: Error test trial
         Given the application has been launched
         And I start a new application
         And I select Standard rules as the permit type
+        And I select Car and vehicle dismantling as the permit category
+        And I select SR2015 No 13 as the permit number
         And I select none as the permit holder
```
Comment (StuAA78) [inline features/test_error.feature]: Should this be `And I enter my permit holder details for none` ?

File: features/test_error.feature [hunk 2]
```diff
@@ -15,6 +17,8 @@ Feature: Error test trial
         Given the application has been launched
         And I start a new application
         #And I select Standard rules as the permit type
+        And I select Car and vehicle dismantling as the permit category
+        And I select SR2015 No 13 as the permit number
         And I select none as the permit holder
```
Comment (StuAA78) [inline features/test_error.feature]: Should this be `And I enter my permit holder details for none` ?

**Review by StuAA78 — COMMENTED** [review]
See comments on `features/test_error.feature`

**Review by amrutaben — APPROVED** [review]

File: features/test_error.feature (hunk 2 above)
Comment (ajaxscape) [reply features/test_error.feature]: Changed as requested

File: features/test_error.feature (hunk 1 above)
Comment (ajaxscape) [reply features/test_error.feature]: Changed as requested

**Pull request merged**